agent/04_agent_configure.sh

```diff
@@ -83,8 +83,14 @@ metadata:
   name: test-agent-cluster-install
   namespace: cluster0
 spec:
+EOF
+if [[ "$AGENT_CONFIG" == "COMPACT_IPV4" || "$AGENT_CONFIG" == "COMPACT_IPV6" || "$AGENT_CONFIG" == "HA_IPV4" || "$AGENT_CONFIG" == "HA_IPV6" ]]; then
```

Comment:
We can put below single condition to check whether it is SNO or not, this will help to avoid adding multiple conditions in future.
```suggestion
if [[ ! "$AGENT_CONFIG" =~ "SNO" ]]; then
```

Reply:
I would like to keep the check based on NUM_MASTERS as already exists today. Moreover, this check will fail when AGENT_CONFIG ( aka E2E_TEST_SCENARIO) is incorrectly set to just HA